### PR TITLE
Restored isWeeks variable for ephemeral messages

### PR DESCRIPTION
**Pull request by nicorsm**
## What's new in this PR?

Restored `isWeeks` variable from UI project.

### CODE REVIEW

File: Source/Model/Conversation/ZMConversation+Ephemeral.swift [hunk 1]
```diff
@@ -141,6 +142,10 @@ public extension MessageDestructionTimeoutValue {
     var isDays: Bool {
         return rawValue >= 86400
```
Comment (daehn) [inline Source/Model/Conversation/ZMConversation+Ephemeral.swift]: Maybe worth changing is Days to check `86400..< 604800 ~= rawValue` in that case.

**Comment by nicorsm** [comment]
@zenkins release minor

**Comment by vytis** [comment]
The reason why magic phrase didn't work - "3 of 4 required status checks are expected.". Basically we can't merge until we have circleci passing. I will turn it off for now and try again!

**Comment by nicorsm** [comment]
@zenkins release minor

**Comment by daehn** [comment]
@zenkins release minor

**Pull request merged**